lms/static/js/spec/edxnotes/views/shim_spec.js

```diff
@@ -233,5 +233,25 @@ define([
                 expect(mockViewer.element.appendTo).toHaveBeenCalledWith(annotators[0].wrapper);
             });
         });
+
+        describe('TagsPlugin', function () {
```

Comment:
@cptvitamin The issue isn't about accessibility itself-- it's the way that the init methods are called on the annotator plugins/editors. I couldn't quite figure it out, but it seemed that the initialization may not be synchronous (or may require some triggers that our unit test setup does not do).